nexus/library/qmcpack_result_analyzers.py

```diff
@@ -67,13 +67,20 @@ def __init__(self,input,opts,energy_weight=None,variance_weight=None,nindent=0):
             else:
                 cv = 0
             #end if
+            if ce is None:
+                ce = 0
+            #end if
             if ew==None:
                 ew = ce
             #end if
             if vw==None:
                 vw = cv
             #end if
         #end if
+        if abs(ew)<tol and abs(vw)<tol:
```

Comment:
This fixes my issue but I don't like the logic blocks and tol.
if curv,crv,ce are not specified at all, put the qmcpack default.
if any of them is specified, put zero in non specified ones and do necessary checks (sum=1)